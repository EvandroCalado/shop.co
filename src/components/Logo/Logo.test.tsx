import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { Logo } from '.';

describe('<Logo />', () => {
  it('should render with default value', () => {
    render(<Logo />);

    expect(screen.getByRole('link')).toBeInTheDocument();
    expect(screen.getByRole('link')).toHaveAttribute('href', '/');
    expect(
      screen.getByRole('heading', { name: 'shop.co' }),
    ).toBeInTheDocument();
  });

  it('should render a snapshot', () => {
    const { container } = render(<Logo />);

    expect(container).toMatchSnapshot();
  });
});
