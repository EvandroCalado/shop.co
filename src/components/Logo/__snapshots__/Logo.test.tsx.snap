// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Logo /> > should render a snapshot 1`] = `
<div>
  <a
    class="text-2xl font-black uppercase sm:block"
    href="/"
  >
    <h1>
      shop.co
    </h1>
  </a>
</div>
`;
